sdk/rust/oak/src/grpc/mod.rs

```diff
@@ -168,7 +168,11 @@ impl<T: ServerNode> crate::Node<Invocation> for T {
 
 /// Encapsulate a protocol buffer message in a GrpcRequest wrapper using the
 /// given method name.
-pub fn encap_request<T: protobuf::Message>(req: T, method_name: &str) -> Option<GrpcRequest> {
+pub fn encap_request<T: protobuf::Message>(
+    req: T,
+    req_type_url: Option<&str>,
```

Comment:
Would it be simpler to just pass in the `Any` message, and have a convenience builder for that as a separate method?